ci/ray_ci/container.py

```diff
@@ -10,31 +10,48 @@
 DOCKER_ECR = "029272617770.dkr.ecr.us-west-2.amazonaws.com"
 DOCKER_REPO = "ci_base_images"
 DOCKER_TAG = f"oss-ci-build_{os.environ.get('BUILDKITE_COMMIT')}"
+ROOT_DIR = "/ray/ci/ray_ci"
 
 
 def run_tests(
+    team: str,
     test_targets: List[str],
-    pre_run_commands: List[str],
     parallelism,
 ) -> bool:
     """
-    Run tests parallelly in docker. Return whether all tests pass.
+    Run tests parallelly in docker.  Return whether all tests pass.
     """
     chunks = chunk_into_n(test_targets, parallelism)
-    # Run tests in parallel. Currently, the logs are also printed in parallel.
-    # TODO(can): We can use a queue to print the logs in order.
-    runs = [_run_tests_in_docker(chunk, pre_run_commands) for chunk in chunks]
+    _setup_test_environment(team)
+    runs = [_run_tests_in_docker(chunk) for chunk in chunks]
     exits = [run.wait() for run in runs]
     return all(exit == 0 for exit in exits)
 
 
-def _run_tests_in_docker(
-    test_targets: List[str],
-    pre_test_commands: List[str],
-) -> subprocess.Popen:
-    bazel_command = ["bazel", "test", "--config=ci"] + test_targets
-    script = "\n".join(pre_test_commands + [" ".join(bazel_command)])
-    return subprocess.Popen(_docker_run_bash_script(script))
+def _setup_test_environment(team: str) -> None:
+    env = os.environ.copy()
+    env["DOCKER_BUILDKIT"] = "1"
+    subprocess.run(
```

Comment:
use `check_call(..., stderr=sys.stderr, stdout=sys.stdout)` ?

this makes sure that when the command fails, it will throw an exception and halt the run.